specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -904,6 +904,10 @@
           "$ref": "#/definitions/WorkspaceCustomParameterType",
           "description": "The type of variable that this is"
         },
+        "entities": {
+          "$ref": "#/definitions/ManagedServicesEncryptionEntities",
```

Comment:
We need to add a node under entities for "managedServices"

Reply:
Fixed.